system/HTTP/RedirectException.php

```diff
@@ -0,0 +1,3 @@
+<?php namespace CodeIgniter\HTTP;
+
+class RedirectException extends \Exception {}
```

Comment:
Sorry. We'd like to make each of the core elements, like the HTTP module, or the Debug, or Router, etc, so that they can be used as standalone packages. While it's not a super-priority, it is something on our list that we need to consider as we develop. So we need to keep Exceptions available with the code that needs it.

Which is fine, but I hate having to load a single file just to create a new exception. I'd almost make a special case for defining the Exception within one of the class files.